### PR TITLE
dashboard: make color, font and line-height adjustments for visual consistency

### PR DESCRIPTION
**Pull request by Zaryafaraj**
We are a focused team with a product roadmap, so we can't guarantee that a pull request will receive the attention it might deserve.

However, if you've found a way to help out the Chain Core community by submitting a bugfix, we'll do our best to work with you to get your code merged. Please use the following process to contribute:

  1. Open a new issue describing the bug or feature
  2. Receive feedback from Chain staff
  3. Collaborate with Chain staff around the design of the code
  4. Write the patch adhering to Chain's Code Style Guide: https://github.com/chain/chain/blob/main/docs/internal/styleguide.md
  5. Complete legal requirements for Chain's Contributor License

If you have questions about how to contribute, please visit the Chain developer support center: https://support.chain.com

### CODE REVIEW

**Comment by Zaryafaraj** [comment]
@dominic PTAL

**Comment by dominic** [comment]
General feedback: what is the meaning behind the font sizes here? This adds font-sizes of 13px, 15px and 16px (and a line-height of an unspecified ratio) - do these have any values in a type hierarchy that we can make clearer with variables instead?

File: dashboard/src/features/transactions/components/Summary/Summary.scss [hunk 1]
```diff
@@ -27,13 +27,15 @@
   background: none;
 }
 
-.odd td {
-  background: $background-emphasis-color;
+td {
+  border-bottom: 1px solid #f2f2f2;
```
Comment (dominic) [inline dashboard/src/features/transactions/components/Summary/Summary.scss]: We just removed a color of #f2f2f2 on line https://github.com/chain/chain/pull/162/files#diff-027ae693252442f625c31b3c345afdd3L13 - should this be `$background-content-color` instead?

**Comment by Zaryafaraj** [comment]
@dominic PTAL

File: dashboard/src/assets/styles/resources.scss [hunk 2]
```diff
@@ -15,6 +15,7 @@ $background-emphasis-color: #fafafa;
 
 $border-color: #dadee2;
 $border-strong-color: #cccccc;
+$border-light-color: #f2f2f2;
```
Comment (dominic) [inline dashboard/src/assets/styles/resources.scss]: If we're going to use the same value as `$background-content-color`, let's use this instead:

```sass
$border-light-color: $background-content-color;
```

File: dashboard/src/assets/styles/_bootstrap-overrides.scss [hunk 3]
```diff
@@ -1,6 +1,15 @@
 $grid-gutter-width: 30px;
 
-$font-size-base: 16px;
+$font-size-caps: 13px;
+$font-size-code: 13px;
```
Comment (dominic) [inline dashboard/src/assets/styles/_bootstrap-overrides.scss]: Same re: reusing variable sizes. Let's assign variable names when two variables have the same value, e.g.:

```sass
$font-size-caps: 13px;
$font-size-code: $font-size-caps;
...
$font-size-nav: 16px;
$font-size-form-section-title: $font-size-nav;
...
$font-size-section-title: 18px;
$font-size-btn-lg: $font-size-section-title;
```

File: dashboard/src/features/transactions/components/Summary/Summary.scss [hunk 4]
```diff
@@ -27,13 +27,15 @@
   background: none;
 }
 
-.odd td {
-  background: $background-emphasis-color;
+td {
```
Comment (dominic) [inline dashboard/src/features/transactions/components/Summary/Summary.scss]: If we're removing `.odd` row styles, we need to also remove it from the markdown. If that's the intention, I'll add an updated the JSX file to this PR.

**Pull request merged**